.travis.yml

```diff
@@ -7,13 +7,16 @@ services:
 matrix:
   include:
     - env: TO_TEST=build
-    - env: TO_TEST=e2e
+    - env: TO_TEST=ephemeral
+    - env: TO_TEST=pyspark-templates
+    - env: TO_TEST=java-templates
+    - env: TO_TEST=scala-templates
   fast_finish: true
 
 before_install:
 ## add insecure-registry and restart docker
 - |
```

Comment:
this just my opinion but removing line 18 and adding if condition for every line make the final travis logs much easier to debug